threeML/test/test_basic.py

```diff
@@ -288,3 +288,27 @@ def test_gbm_workflow():
     dl_files = p.glob("*_bkg.h5")
 
     [x.unlink() for x in dl_files]
+
+
+
+
+def test_uncertainty_formatter():
```

Comment:
I guess we could explicitly test the return values here, but that might be overkill.

Reply:
good point @henrikef I will add that and then merge